README.md

```diff
@@ -81,7 +81,7 @@ For auto-syncing `data/` files, we use Gulp. This sets up a watcher for automati
 ```shell
 npm install --global gulp-cli
 npm install --save-dev gulp gulp-watch gulp-changed
-run the
+# run the file watcher
```

Comment:
For this file, take upstream instead of HEAD since I just updated the entire README and fixed them